tests/models/whisper/test_feature_extraction_whisper.py

```diff
@@ -213,7 +207,8 @@ def _load_datasamples(self, num_samples):
 
         return [x["array"] for x in speech_samples]
 
-    def test_integration(self):
+    @require_torch
+    def test_torch_integration(self):
```

Comment:
Here we show we get the same output values with a numpy backend as with a torch one